CONTRIBUTING.md

```diff
@@ -0,0 +1,117 @@
+# Contributing to Holoscan CLI
+
+## Prerequisites
+
+To contribute to the Holoscan CLI repository, you will need to install the following dependencies:
+
+- [poetry](https://python-poetry.org/docs/#installation)
+
+### Workflow
+
+1. Developers must first [fork](https://help.github.com/en/articles/fork-a-repo) the [upstream](https://github.com/nvidia-holoscan/holoscan-cli) Holoscan CLI repository.
+
+1. Git clone the forked repository and push changes to the personal fork.
+
+```bash
+git clone https://github.com/YOUR_USERNAME/YOUR_FORK.git holoscan-cli
+# Checkout the targeted branch and commit changes
+# Push the commits to a branch on the fork (remote).
+git push -u origin <local-branch>:<remote-branch>
+```
+
+3. Once the code changes are staged on the fork and ready for review, please [submit](https://help.github.com/en/articles/creating-a-pull-request) a [Pull Request](https://help.github.com/en/articles/about-pull-requests) (PR) to merge the changes from a branch of the fork into a selected branch of upstream.
+
+- Exercise caution when selecting the source and target branches for the PR.
+- Creation of a PR creation kicks off the code review process.
+
+4. Holoscan CLI maintainers will review the PR and accept the proposal if changes meet Holoscan CLI standards.
```

Comment:
Are we clear to accept public PR contributions? @jjomier

Reply:
I have upated both README and